packages/plugins/plugin-adjust/src/AdjustPlugin.tsx

```diff
@@ -66,6 +66,7 @@ export class AdjustPlugin extends DestinationPlugin {
         adjustConfig.setDelayStart(delayTime);
       }
     }
+    this.registeredCallback = true;
```

Comment:
You need to put this inside the condition that checks if the callback has been registered (after registering the callback)